app/controllers/events/view/tickets/orders/list.js

```diff
@@ -74,6 +74,27 @@ export default Controller.extend({
         .finally(() => {
           this.set('isLoading', false);
         });
+    },
+    async resendConfirmation(order) {
+      let payload = {};
+      try {
+        payload = {
+          'data': {
+            'order' : order.identifier,
+            'user'  : this.authManager.currentUser.email
+          }
+        };
+        await this.loader.post('orders/resend-email', payload);
+        this.notify.success(this.l10n.t('Email confirmation has been sent to attendees successfully'));
+      } catch (error) {
+        console.log(error);
+        const errorMessage = error.message || error.errors[0].detail;
```

Comment:
Give first priority to `error.errors[0].detail`. Also, what if server is sending multiple errors. You should not hard code to display only the first.

Reply:
@niranjan94 Understood.

Reply:
@niranjan94 Understood